commands/post-install.ts

```diff
@@ -32,6 +33,12 @@ export class PostInstallCommand implements ICommand {
 
 			this.checkSevenZip().wait();
 			this.$commandsService.tryExecuteCommand("autocomplete", []).wait();
+
+			var sysInfo = this.$sysInfo.getSysInfo();
+			if (!sysInfo.adbVer) { console.warn("Cannot find adb in the path. The built-in one will be used, which may be incompatible with the latest Android SDK or Genymotion"); }
+			if (!sysInfo.antVer) { console.warn("Cannot find ant. You cannot build Android projects."); }
```

Comment:
appbuilder is also calling PostInstallCommand and these NativeScript specific messages will scare our users. They should be moved to nativescript-cli only. 
(or I'm totally missing something here)